CMakeLists.txt

```diff
@@ -46,7 +46,7 @@ endif()
 # Add custom functions / macros
 list(APPEND CMAKE_MODULE_PATH ${CMAKE_CURRENT_SOURCE_DIR}/cmake)
 
-find_package(YCM REQUIRED)
```

Comment:
Can you bump the robotology-superbuild YCM tag before the superbuild points to this commit? Otherwise this will fail.

Reply:
Yep thanks, Travis already reminded me :sweat_smile: